src/app/products/brands/brands.component.html

```diff
@@ -0,0 +1,3 @@
+<div style="float: right; position: absolute; top: 0%; left: 25%; font-size: 0.9em;">
```

Comment:
you should create a class for this inline styles, so that they can still be reusable